### PR TITLE
TTT-26: Makes has_won subfunctions private. Updates README.md.

### PR DESCRIPTION
**Pull request by billlanyon**

### CODE REVIEW

File: com/bl/ttt/tictactoe.py [hunk 1]
```diff
@@ -32,27 +32,28 @@ def _is_cell_empty(self, move):
     def make_valid_move(self, move):
         self.cells[move.cell_chosen] = move.player_id
 
-    def is_any_row_complete(self, player_id):
+    def has_won(self, player_id):
+        if self._is_any_row_complete(player_id) or self._is_any_column_complete(player_id) or \
+                self._is_any_diagonal_complete(player_id):
+            return self
```
Comment (BasilBibi) [inline com/bl/ttt/tictactoe.py]: Should be return true ?
also good to put in an else return false

Comment (BasilBibi) [reply com/bl/ttt/tictactoe.py]: I know that None evaluates to False in truthiness but lets be explicit about it

Comment (billlanyon) [reply com/bl/ttt/tictactoe.py]: Updated.

File: com/bl/ttt/tictactoe.py [hunk 2]
```diff
@@ -32,27 +32,30 @@ def _is_cell_empty(self, move):
     def make_valid_move(self, move):
         self.cells[move.cell_chosen] = move.player_id
 
-    def is_any_row_complete(self, player_id):
+    def has_won(self, player_id):
+        if self._is_any_row_complete(player_id) or self._is_any_column_complete(player_id) or \
```
Comment (BasilBibi) [inline com/bl/ttt/tictactoe.py]: Ok looking at this again i see you could do this : 
`return self._is_any_row_complete(player_id) or self._is_any_column_complete(player_id) or ...`

Comment (BasilBibi) [reply com/bl/ttt/tictactoe.py]: Sorry to mess you about - I am a bit distracted today !

**Review by BasilBibi — APPROVED** [review]
Small change in has_won

File: com/bl/ttt/tictactoe.py (hunk 2 above)
Comment (billlanyon) [reply com/bl/ttt/tictactoe.py]: Done.

**Pull request merged**